api/routes.go

```diff
@@ -2,13 +2,15 @@ package api
 
 import (
 	"github.com/go-macaron/binding"
+	"github.com/go-macaron/gzip"
 	"github.com/raintank/metrictank/api/middleware"
 	"github.com/raintank/metrictank/api/models"
 	"gopkg.in/macaron.v1"
 )
 
 func (s *Server) RegisterRoutes() {
 	r := s.Macaron
+	r.Use(gzip.Gziper())
```

Comment:
and I guess if gzipping somehow becomes an issue (in terms of cpu or memory used) we would see it in a profile at some point.